deploy/build/Dockerfile.pr.amd64

```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1
+
+FROM public.ecr.aws/debian/debian:bookworm-slim as runtime
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates
```

Comment:
Pretty sure that debian base image already has `ca-certificates`

Reply:
No :) it doesn't actually.
```
docker run --rm -it public.ecr.aws/debian/debian:bookworm-slim /bin/bash
$ ls -al /etc/ssl/certs
$ apt update -y && apt install -y ca-certificates
$ ls -al /etc/ssl/certs
```

Reply:
Learn something every day! 😂